config/locales/en.yml

```diff
@@ -218,6 +218,10 @@ en:
       usually indicates that an article follow Wikipedia's conventions for the
       structure of a good article, but it does not take into account the content
       itself. (These scores are only available for English Wikipedia articles.)
+    items:
```

Comment:
There are three different possible names for "Article" which we want to support:
* "Article" for wikipedia
* "Item" for wikidata
* "Page" for anything else

I think instead of having these be within the `articles` i18n branch, we should follow the same pattern as with `courses` and `courses_generic`. To keep them near each other in the file and match the pattern of `courses_generic`, this might mean the new branch keys are `articles_wikidata` and `articles_generic`. So we'd add a new set of messages as the same level as the `articles` branch, and switch which branch to use based on the wiki prefix.